src/controllers/controller.radar.js

```diff
@@ -85,43 +72,122 @@ module.exports = DatasetController.extend({
 		var custom = point.custom || {};
 		var dataset = me.getDataset();
 		var scale = me.chart.scale;
-		var pointElementOptions = me.chart.options.elements.point;
 		var pointPosition = scale.getPointPositionForValue(index, dataset.data[index]);
+		var x, y;
+
+		var options = me._resolvePointOptions(point, index);
+
+		x = reset ? scale.xCenter : pointPosition.x;
+		y = reset ? scale.yCenter : pointPosition.y;
+
+		// Utility
+		point._scale = scale;
+		point._options = options;
+		point._datasetIndex = me.index;
+		point._index = index;
+
+		// Desired view properties
+		point._model = {
+			x: x, // value not used in dataset scale, but we want a consistent API between scales
+			y: y,
+			skip: custom.skip || isNaN(x) || isNaN(y),
+			// Appearance
+			radius: options.radius,
+			pointStyle: options.pointStyle,
+			rotation: options.rotation,
+			backgroundColor: options.backgroundColor,
+			borderColor: options.borderColor,
+			borderWidth: options.borderWidth,
+			tension: resolve([custom.tension, dataset.lineTension, me.chart.options.elements.line.tension]),
```

Comment:
Shouldn't this be: `tension: line._model.tension`?

Reply:
`line._model.tension` first checks `meta.dataset.custom.tension` while `point._model.tension` checks `meta.data[]._model.custom.tension`, otherwise the test will fail. This is one of the differences from the line chart which doesn't use `meta.data[]._model.custom.tension`.

Reply:
It seems in line chart, tension applies to whole line, while in radar you can have different tensions per point. Only the tension in point is used if I understood correctly.

Reply:
That's correct.

Reply:
I think we should make that logic consistent between the line and radar charts. If `tension` is configurable per point (and if that makes sense), then maybe the line chart should adopt the same logic (which would make both controllers even more similar).

I also agree that `custom` should be deprecated in favor of scriptable options. But doesn't that mean that the `lineTension` should be scriptable as well?

Reply:
To be honest, I don't see any use case where each point has a different line tension, and actually we haven't received any request for it for a line chart. The radar controller is using `point.custom.tension`, but I'm not sure if anyone is using it. Shouldn't we just use the same logic as the line controller, or is that a braking change we should avoid?

Reply:
Per point tension seems quite unusable as it's implemented now so I would vote for a unique tension for the whole line in radar charts. But this behavior exists since forever (06d55e3501c) and I would not be surprised if someone found a use case for a per point tension using `custom`. @etimberg thoughts?

Since `custom` is intended to die, we could simply port that `point.custom` logic in the line chart in order to be able to refactor both controllers without breaking change, but we would not make `tension` scriptable.

@nagix what do you think?

Reply:
Neither `custom` nor per-point tension are documented, so a user would have had to find two undocumented features to use it. My vote would be for the simpler implementation. But I'm happy with whatever the majority decide

Reply:
> Since `custom` is intended to die, we could simply port that `point.custom` logic in the line chart in order to be able to refactor both controllers without breaking change, but we would not make `tension` scriptable.

I'm fine with this. There is nearly zero chance of `line.custom.tension` being used instead of `dataset.tension`.

Reply:
That is quite an old PR. Not sure why I had an idea of different tensions per point but I believe it was to control how certain points influence the bezier curve (before we had monotonic interpolation).

I am on board with @simonbrunel's proposal

Reply:
Ported per-point `custom.tension` logic to the line controller.